guide/popular-topics/builders.md

```diff
@@ -0,0 +1,132 @@
+# Builders
+
+Discord.js provides a `builders` package which contains a variety of builders and utilities you can use when writing your Discord bot.
+To install the package run `npm i @discordjs/builders` in your terminal.
+
+## Formatters
+
+Formatters are a set of utility functions which format input strings into the given format.
+
+### Basic Markdown
+
+The Formatters provide functions to format strings into all the different Markdown styles supported by Discord.
+
+```js
+const { Formatters } = require('@discordjs/builders');
+const string = 'Hello!';
+
+const bold = Formatters.bold(string);
+const italic = Formatters.italic(string);
+const strikethrough = Formatters.strikethrough(string);
+const spoiler = Formatters.spoiler(string);
+const quote = Formatters.quote(string);
+const blockquote = Formatters.blockQuote(string);
+const underscore = Formatters.underscore(string);
+```
+
+### Links
+
+There are also two methods to format hyperlinks. `Formatters#hyperlink()` will format the URL into a masked markdown link and `Formatters#hideLinkEmbed()` will wrap the URL in `<>`, preventing it from embedding.
+
+```js {2,4-5}
+const { Formatters } = require('@discordjs/builders');
+const url = 'https://discord.js.org/';
+
+const link = Formatters.hyperlink(url);
+const hiddenembed = Formatters.hideLinkEmbed(url);
+```
+
+### Codeblocks
+
+You can use `Formatters#inlineCode()` and `Formatters#codeBlock()` to turn a string into an inline codeblock or a regular codeblock with or without syntax highlighting.
+
+```js {2,4-6}
+const { Formatters } = require('@discordjs/builders');
+const jsstring = 'const value = true;';
+
+const inline = Formatters.inlineCode(jsstring);
+const codeblock = Formatters.codeBlock(jsstring);
+const highlighted = Formatters.codeBlock('js', jsstring);
+```
+
+### Timestamps
+
+With `Formatters#time()` you can format UNIX timestamps and dates into a Discord timestring.
+
+```js {2,4-5}
+const { Formatters } = require('@discordjs/builders');
```

Comment:
There is no `Formatters` class, you need to export each function you'd like to use individually ([eg](https://github.com/discordjs/builders/blob/main/tests/messages.test.ts#L1-L20))